app/v1/models/errors/mtdErrors.scala

```diff
@@ -47,10 +47,12 @@ object RuleTaxYearNotSupportedError
 object RuleIncorrectOrEmptyBodyError extends MtdError("RULE_INCORRECT_OR_EMPTY_BODY_SUBMITTED", "An empty or non-matching body was submitted")
 
 object RuleTaxYearRangeExceededError
-    extends MtdError("RULE_TAX_YEAR_RANGE_EXCEEDED", "Tax year range exceeded. A tax year range of one year is required.")
+    extends MtdError("RULE_TAX_YEAR_RANGE_EXCEEDED", "Tax year range exceeded. A tax year range of one year is required")
 
 object RuleDateRangeInvalidError extends MtdError("RULE_DATE_RANGE_INVALID", "The specified date range is invalid")
 
+object RuleInsolventTraderError extends MtdError("RULE_INSOLVENT_TRADER", "Unable to return data as trader is insolvent")
+
```

Comment:
Looks like some other changes got dragged into this PR.

Reply:
ah yeah I'll remove that one sec